### ISSUE TITLE
"and others" problem with just single explicit author and others

### ISSUE THREAD

**Issue by fabianrost84** [plk/biblatex-apa#20]
I have the following database entry (from https://inspirehep.net/record/1315584/export/hx):
```
@article{Agashe:2014kda,
  title = {Review of {{Particle Physics}}},
  volume = {C38},
  doi = {10.1088/1674-1137/38/9/090001},
  timestamp = {2016-04-13T16:16:07Z},
  journaltitle = {Chin. Phys.},
  author = {Olive, K. A. and {others}},
  date = {2014},
  pages = {090001},
  collaboration = {Particle Data Group},
  slaccitation = {%%CITATION = CHPHD,C38,090001;%%}
}
```
when I insert a citation using `\cite{Agashe:2014kda}` I would expect to get `Olive et al., 2014` but I get `Olive, 2014`. Is `Olive, 2014` correct or is that a bug?

An example (assuming the bib file is named test.bib) to reproduce the problem:

```
\documentclass[a4paper,10pt]{article}
\usepackage[utf8]{inputenc}
\usepackage{csquotes}
\usepackage[american]{babel}
\usepackage[style=apa,backend=biber]{biblatex}
\DeclareLanguageMapping{american}{american-apa}
\addbibresource{test.bib}

\begin{document}

\cite{Agashe:2014kda}

\end{document}
```

**Comment by plk** [comment]
It's a bug, thank you. APA mandates that "et al" only covers more than one truncation but I think we have to assume that "and others" is more than one. I will look at fixing this in v6.9 shortly and update here.

**Comment by plk** [comment]
This should be fixed in the development version v6.9 which you can find here:

https://www.dropbox.com/s/nh0ng52ym2bhpne/biblatex-apa.tgz?dl=0

You will need to be using the latest development version of biblatex (3.4) from sourceforge and also the latest development version of biber (2.5) also on sourceforge.
